app/controllers/manager/venues_controller.rb

```diff
@@ -39,4 +51,14 @@ def require_user
       end
     end
 
+    def require_correct_manager
```

Comment:
So much cleaner. I went with the top one as i felt it was more readable and cleaner.